internal/commands/result.go

```diff
@@ -690,6 +690,8 @@ func enhanceWithScanSummary(summary *wrappers.ResultSummary, results *wrappers.S
 		}
 	}
 
+	summary.TotalIssues = summary.SastIssues + summary.ScaIssues + summary.KicsIssues + summary.GetAPISecurityDocumentationTotal()
```

Comment:
can you explain what the bug was and how it fixed it?

Reply:
They added scstotalIssues to  summary.TotalIssues in function printConsoleSummary.
this behavior is wrong because when FF is off or scs engine was not selected it will add -1 to the results.
I moved this update to its correct place